specification/schemas/HookLog.json

```diff
@@ -0,0 +1,121 @@
+{
+  "type": "object",
+  "required": [
+    "attributes",
+    "relationships"
+  ],
+  "properties": {
+    "attributes": {
+      "type": "object",
+      "required": [
+        "resource_diff",
+        "errors"
+      ],
+      "additionalProperties": false,
+      "properties": {
+        "resource_diff": {
+          "type": "object",
+          "example": {
+            "relationships" : {
+              "service": {
+                "data": {
+                  "id" : "7b808eee-bf1c-40cd-98f2-3c335a06417e"
+                }
+              }
+            }
+          },
+          "description": "The difference between the resource before and after the hook has run."
+        },
+        "errors": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "example": "Applying the hook failed because the selecting service was not available for the shipment contract."
+          },
+          "description": "An array of errors encountered when trying to run this hook."
+        }
+      }
+    },
+    "relationships": {
+      "type": "object",
+      "required": [
+        "resource",
+        "hook"
+      ],
+      "properties": {
+        "resource": {
+          "type": "object",
+          "required": [
+            "data"
+          ],
+          "additionalProperties": false,
+          "properties": {
+            "data": {
+              "type": "object",
+              "required": [
+                "type",
+                "id"
+              ],
+              "properties": {
+                "type": {
+                  "type": "string",
+                  "example": "shipments"
+                },
+                "id": {
+                  "type": "string",
+                  "format": "uuid",
+                  "example": "7b808eee-bf1c-40cd-98f2-3c335a06417e"
+                }
+              }
+            },
+            "links": {
+              "readOnly": true,
+              "type": "object",
+              "required": [
+                "related"
+              ],
+              "additionalProperties": false,
+              "properties": {
+                "related": {
+                  "type": "string",
+                  "format": "url",
+                  "example": "$API_HOST/shipments/5c868557-0827-4d21-a7f4-9820f01769f4"
+                }
+              }
+            }
+          },
+          "example": {
+            "data": {
+              "type": "shipments",
+              "id": "9cdf86e8-333f-4ed9-bb31-4935c780c947"
+            }
+          }
+        },
+        "hook": {
+          "$ref": "#/components/schemas/HookLogRelationship",
+          "example": {
+            "data": {
+              "type": "organizations",
```

Comment:
Doesn't the HookLogRelationship schema already specify an example? And also `"type": "organizations"`??